kuma/attachments/views.py

```diff
@@ -59,6 +54,7 @@ def stream_raw_file(*args):
                 response['Content-Length'] = rev.file.size
             except OSError:
                 pass
+            response['Last-Modified'] = convert_to_http_date(rev.created)
```

Comment:
👍 conversion to ``convert_to_http_date``. I'm guessing that the ``ETag`` skips streaming responses, so this is still needed.

``convert_to_utc`` can't handle ambiguous dates:

```python
import datetime
from kuma.attachments.utils import convert_to_utc
convert_to_utc(datetime.datetime(2017, 11, 5, 1, 8, 42))
```

That feels like follow-on work on [bug 1173189](https://bugzilla.mozilla.org/show_bug.cgi?id=1173189), and moving date manipulations into ``kuma/core/utils.py``.

Reply:
Good point about streaming responses (we could have removed this code that sets the `Last-Modified` header if it wasn't a streaming response), and good catch about `content_to_utc` not handling ambiguous dates! I would never have thought of that until it failed. I guess the solution is to follow what you did for the publication date for `DocumentsFeed`, and assume DST if the timezone is naive? I'll do that in a follow-on PR.

Reply:
It was on my mind because of PR #4621. I'd use the same strategy of assuming DST and being half-wrong rather than raise an ISE.

The function may be slightly different because that one converts to server timezone, while this one wants UTC. Maybe! I'd have to look at ``get_current_timezone`` vs ``get_default_timezone``.

Getting to ``USE_TZ=1`` is on my list, and I think centralizing the timezone stuff in ``kuma.core.utils`` would be a good step. This stuff may be a shared function, or two close functions in the same file